index.html

```diff
@@ -243,10 +244,91 @@ <h4 class="media__body__heading">Brenton Cleeland</h4>
                                     </ul>
                                 </div>
                             </div>
+                        </article> -->
+
+                      <article class="past-speaker Grid-cell">
+                            <div class="media">
+                                <div class="media__figure">
+                                    <img src="https://pbs.twimg.com/profile_images/1755583699/me-smaller_400x400.JPG" style="width:100px;height:100px;"/>
```

Comment:
The `style` here should be removed and added to the `_objects.media.scss` file if required (though it should already be settings the width to 80px).